chsdi/tests/integration/test_features_service.py

```diff
@@ -270,7 +270,7 @@ def test_extendedhtmlpopup_valid_with_callback(self):
         self.assertEqual(resp.content_type, 'application/javascript')
 
     def test_extendedhtmlpopup_noinfo(self):
-        self.testapp.get('/rest/services/ech/MapServer/ch.bafu.bundesinventare-bln/362/extendedHtmlPopup', status=404)
+        self.testapp.get('/rest/services/ech/MapServer/ch.bafu.bundesinventare-bln/1/extendedHtmlPopup', status=404)
```

Comment:
because you changed the table structure all these tests are failing on main. The day of the deploy we'll have to deploy without the tests if you don't leave the table